src/content/docs/tutorial-innovation-and-growth/release-quality-mgmt.mdx

```diff
@@ -0,0 +1,220 @@
+
+Prior to starting any custom integration work, you should determine if any out-of-box integrations exist that meet your goals.
+    </Step>
+    <Step>
+### Implement dashboards [#implement-dashboards]
+
+Our <InlinePopover type="dashboards" /> are the primary drivers of the quality improvement process. They'll show KPIs and trends so you can identify and prioritize your improvement efforts. Sample dashboards can be [found in our observability maturity resource center on GitHub](https://github.com/newrelic/oma-resource-center).
+
+The information displayed in the dashboards depends on your development toolchain, so you'll need to [customize your dashboard](https://developer.newrelic.com/collect-data/monitor-your-application/query-custom-data/) to your exact specifications.          
+    </Step>
+    <Step>
+### Establish a baseline [#establish-baseline]
+
+Because you need enough data to form a baseline before you can [perform the initial enablement](#perform-enablement-one), you must establish your baselines that consist of a sample of development activity. Normally, this will be a minimum of two weeks, but it can be up to six weeks depending on your current development pace. One easy way to do this is to align your baseline collection and evaluation cycle with your Agile sprints, if applicable.
+
+You should periodically ensure that event data is accumulating as expected in New Relic while you establish your baselines.        
+    </Step>
+    <Step>
+### Meet with your team [#meet-with-team]
+
+After establishing your baselines, you'll introduce development teams and other stakeholders to the collected data and the ongoing continuous improvement process you'll be following.
+
+The process consists of four activities:
+1. **Introduce the concepts of trunk-based development**: You and the stakeholders will review the core concepts of truck-based development, identify where your current practices differ, and then create strategies to implement it.
+2. **Review your release KPIs and trends**: You'll review the release rate and release size and scope KPIs to ensure you're making progress towards implementing trunk-based development. Your goal is to increase your release rate while reducing the size and scope of new releases.
+3. **Review your application KPIs and trends**: Here, you'll review your application's performance and error KPIs to identify and prioritize your efforts towards improving application reliability and performance.
+4. **Make technical recommendations**: Here, you and the relevant stakeholders will identify and review technical recommendations, such as making changes to your release workflows or observability strategies.
+
+You can use the [session template presentation](https://docs.google.com/presentation/d/1FflqMMVejwn6HMjEni8igZh6UR0a3_JbuMcS5xlZiJU/edit?usp=sharing) to keep your process organized.        
```

Comment:
People need permission to open this up, so the link doesn't work. just wanted to point it out in case that wasn't intentional

Reply:
Ooo I hate when that happens. I've chopped out the link and cast it to the shadow realm. Thank you!